spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -211,6 +211,38 @@
       end
     end
 
+    shared_examples 'allows biometric IAL only if sp is authorized' do |biometric_ial|
+      let(:acr_values) { biometric_ial }
+      context "when the IAL requested is #{biometric_ial}" do
+        context 'when the service provider is allowed to use biometric ials' do
+          before do
+            allow_any_instance_of(ServiceProvider).to receive(:biometric_ial_allowed?).
+              and_return(true)
+          end
+          it 'succeeds validation' do
```

Comment:
[nitpick] can you add newlines after the blocks? a little more whitespace helps readability (like [saml_request_validator_spec.rb](https://github.com/18F/identity-idp/pull/10908/files#diff-1ae19fb77f7fdc9210a8ceca1acd200dda13274e03d68601c098cf3401cfb867))